.github/workflows/python-app.yml

```
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```

Maybe have a better test ... also my suggestion is to rename the file main.yaml ... as this should be the main and the only workflow for this project :)

```suggestion
name: Tests
```